### DISCUSSION TITLE
Pull in all metadata

### DISCUSSION THREAD

**Discussion by jhurlbut** [okio-ai/nendo-platform#16] [Ideas]
The comment field is pulled into the library but none of the other existing meta data is added to the track even though it is output in the mp3 conversion. The metadata I have is TKEY, SERATO_PLAYCOUNT, TBPM, EnergyLevel, and TSSE. It would be cool if those were brought in too. Maybe the bpm should pull from the metadata instead of the analysis if it's found on import

**Comment by faradox** [comment]
Related issue https://github.com/okio-ai/nendo/issues/20
